go.mod

```diff
@@ -6,5 +6,6 @@ require (
 	github.com/fatih/color v1.9.0
 	github.com/goware/urlx v0.3.1
 	github.com/pkg/errors v0.8.2-0.20200103112531-6d954f502eb8
+	github.com/stretchr/testify v1.6.1
```

Comment:
Could you please consider using `gotest.tools/v3` instead of `github.com/stretchr/testify`? to have only one testing framework.

Reply:
ok removed